Makefile

```diff
@@ -5,7 +5,7 @@ MOBILE_PKG := $(PKG)/mobile
 BTCD_PKG := github.com/btcsuite/btcd
 LINT_PKG := github.com/golangci/golangci-lint/cmd/golangci-lint
 GOACC_PKG := github.com/ory/go-acc
-GOIMPORTS_PKG := golang.org/x/tools/cmd/goimports
+GOIMPORTS_PKG := github.com/rinchsan/gosimports/cmd/gosimports
```

Comment:
TIL. The only downside I see is that the project may go out of maintenance.

Reply:
True. But it seems the original one wasn't really that well maintained anyway.